Makefile.am

```diff
@@ -186,7 +186,7 @@ include doc/Makefile-doc.am
 include doc/guide/Makefile-guide.am
 include doc/man/Makefile-man.am
 
-noinst_DATA += doc/guide/html/index.html
+noinst_DATA += doc/guide/html
 
 install-doc-local: doc/guide/html/index.html
```

Comment:
Without that change you get errors like:
No rule to make target 'doc/guide/html/index.html', needed by 'all-am'.  Stop.